### DISCUSSION TITLE
Can GNNexplainer node importance be compared column wise that is can we see how single node importance change as output changes

### DISCUSSION THREAD

**Discussion by bojack-horseman91** [pyg-team/pytorch_geometric#9224] [Q&A] [answered]
I have created a csv file containing node importance of all my test samples now I want to know if I can take a single column i.e. importance of a single node accross multiple column. I want to see how the importance vary as my output changes is this a  valid attempt?

**Comment by rusty1s** [comment]
Can you clarify? Do you want to change the input and see how the importance varies? I think fidelity or unfaithfulness metrics are what you are looking for, see https://pytorch-geometric.readthedocs.io/en/latest/modules/explain.html#explanation-metrics.

**Comment by bojack-horseman91** [comment]
No I am more interested to know that correlation between ground-truth and
and node importance. I want to know if this is a valid step to take ? can I
take the correlation between not importance and GroundTruth and make
implications from their relationship?

**Reply by rusty1s** [reply, marked as answer]
Yes, I think that's definitely possible.
